packages/forms/src/UIForm/fields/Text/Text.component.js

```diff
@@ -46,7 +46,7 @@ if (process.env.NODE_ENV !== 'production') {
 			readOnly: PropTypes.bool,
 			title: PropTypes.string,
 			type: PropTypes.string,
-		}),
+		}).isRequired,
```

Comment:
schema is required and none of its component is, isn't it more safe then to rely on a default props set to an empty literral object ?